metagpt/actions/prepare_documents.py

```diff
@@ -36,7 +36,8 @@ def _init_repo(self):
         if not path:
             name = CONFIG.project_name or FileRepository.new_filename()
             path = Path(CONFIG.workspace_path) / name
-
+        else:
+            path = Path(CONFIG.project_path)
```

Comment:
This new logic is wrong and contrary to line 35

Reply:
`CONFIG.project_path` comes from `--project-path` argument, str type, '' or a valid path.

Reply:
I see what you mean, maybe we should use different variable names? The path that precedes and the path that follows is not of the same type